routes/web.php

```diff
@@ -27,4 +29,7 @@
 Route::middleware('auth')->group(function(){
     Route::get('/logout', [LoginController::class, 'logout'])->name('user.logout');
     Route::resource('/worklogs', WorklogController::class)->except(['show', 'destroy']);
+    Route::group(['prefix' => 'admin', 'middleware' => 'admin', 'name' => 'admin.'], function(){
+        Route::resource('/worklogs', AdminWorklogController::class, ['as' => 'admin'])->except(['show']);
```

Comment:
in `resource` route declaration, is `as` necessary ?
and here the `name` in route group is also admin and resource route's name is also admin.
what is the final route name. please verify it